spec/intl.html

```diff
@@ -1626,7 +1626,9 @@ <h1>Temporal.Calendar.prototype.dateFromFields ( _fields_ [ , _options_ ] )</h1>
               1. Let _result_ be ? ISODateFromFields(_fields_, _options_).
             1. Else,
               1. Let _overflow_ be ? ToTemporalOverflow(_options_).
-              1. Set _fields_ to ? PrepareTemporalFields(_fields_, « *"day"*, *"era"*, *"eraYear"*, *"month"*, *"monthCode"*, *"year"* », « *"day"* »).
+              1. Let _fieldNames_ be CalendarDateFields(_calendar_.[[Identifier]], « *"day"*, *"month"*, *"monthCode"*, *"year"* »).
+              1. Sort _fieldNames_ alphabetically.
+              1. Set _fields_ to ? PrepareTemporalFields(_fields_, _fieldNames_, « »).
```

Comment:
This no longer passes “day” in the second list, is that intended?

Reply:
I intended it provisionally, although I'd like to hear @sffc's feedback on whether that is a change we actually need to make. If "day" is basically required for all calendars, we can reinstate it here. If there are calendars in ICU that might use alternate custom fields to indicate the day, then we shouldn't pass it as required and it was probably incorrect that we did so previously.

Reply:
I agree with the style of not having a blanket requirement for any particular fields in this situation. If there is a required list, that list should come from the calendar, not hard-coded in the spec.